change/@azure-msal-angularjs-2020-08-06-13-11-59-beachball-config.json

```diff
@@ -0,0 +1,8 @@
+{
+  "type": "none",
+  "comment": "Add beachball config",
+  "packageName": "@azure/msal-angularjs",
+  "email": "prkanher@microsoft.com",
```

Comment:
Should we be using team alias or even nugetaad email instead?

Reply:
This is based off of the github account that made PR, it puts the author next to the item in the changelog. Not sure if we want to change this?

Reply:
I assumed this was the email alias that was used for releases. Makes sense to have the author who made the change for changelog purposes.